tests/boardtest.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+git submodule update --init autohat
+cd autohat
+export application_name=`echo $testNode | tr '[:upper:]' '[:lower:]'`
+
+/bin/cat <<EOM >start.sh
```

Comment:
Full path to "cat" here, but not to "git" above. I can't imagine "/bin" to not be in $PATH in a situation when "/usr/bin" is.